lib/tmpl.js

```diff
@@ -165,7 +165,7 @@ var tmpl = (function() {
     return !s ? '' : '(function(v){try{v='
 
         // prefix vars (name => data.name)
-        + (s.replace(re_vars, function(s, _, v) { return v ? '(d.'+v+'===undefined?window.'+v+':d.'+v+')' : s })
+        + (s.replace(re_vars, function(s, _, v) { return v ? '(d.'+v+'===undefined?(typeof window!==\'undefined\'?window.'+v+':global.'+v+'):d.'+v+')' : s })
```

Comment:
this simple change make it work with `node/io.js` where `window` is `undefined` but there is `global`